src/libs/actions/Link.ts

```diff
@@ -42,18 +34,11 @@ function buildOldDotURL(url, shortLivedAuthToken) {
     });
 }
 
-/**
- * @param {String} url
- * @param {Boolean} shouldSkipCustomSafariLogic When true, we will use `Linking.openURL` even if the browser is Safari.
```

Comment:
Can we leave this note in? It might be useful for the future!

Reply:
Added @dangrous 😄